CMakeLists.txt

```diff
@@ -126,40 +126,12 @@ if (NOT (DEFINED UDUNITS_ACTIVE))
 endif()
 
 if(UDUNITS_ACTIVE)
-    find_package(UDUNITS2)
+    find_package(UDUNITS2 REQUIRED)
```

Comment:
Would it help if instead of just `REQUIRED`, I had it error out with a message pointing to the changed documentation and suggested a package installation?